### PR TITLE
Final touches on mobile image create UI

### PR DESCRIPTION
**Pull request by danielagd**
Co-authored-by: Robert Boyd III <git@robertboyd.dev>

### CODE REVIEW

**Comment by rob-3** [comment]
Is this ready to merge?

**Comment by danielagd** [comment]
> Is this ready to merge?

we can merge for the time being. i'm still working on small fixes but they can be in future prs.

File: mobile/src/pages/ImageCreate.tsx [hunk 1]
```diff
@@ -1,56 +1,100 @@
...
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { ActivityIndicator, Text, Pressable, View, ScrollView, SafeAreaView } from "react-native";
 import { imageStyles } from "../data/styles";
 import Button from "../components/Button";
 import Card from "../components/Card";
 import Icon from "react-native-vector-icons/Ionicons";
+import MultiSelect from "../components/MultiSelect";
 import BottomNavBar from "../components/BottomNavBar";
 import TopNavBar from "../components/TopNavBar";
-import SelectableChip from "../components/SelectableChip";
```
Comment (fabriUCF) [inline mobile/src/pages/ImageCreate.tsx:9]: Don't forget to delete this file in the next PR since it's not going to be used.

**Review by fabriUCF — APPROVED** [review]

**Comment by kitorer** [comment]
small issue, on the drop down, you cannot scroll the list unless you are typing in it

File: mobile/src/pages/ImageCreate.tsx [hunk 2]
```diff
@@ -1,56 +1,100 @@
...
+                        selectedItems={selectedStyles}
+                        selectedItemTextColor="#312e81" 
+                        selectedItemIconColor="#312e81" 
+                        tagBorderColor="#312e81"
+                        submitButtonColor="#312e81" 
+                        tagTextColor="#312e81" 
+                        tagRemoveIconColor="#ed4242"
+                        tagContainerStyle={{ backgroundColor: "white" }}
+                        styleTextTag={{ }} />
+                    <Button name="Create" onPress={handleCreate} className="mt-4"/>
```
Comment (kitorer) [inline mobile/src/pages/ImageCreate.tsx:81]: while a image is being created, disable this button to prevent spam calls

**Review by kitorer — APPROVED** [review]

File: mobile/src/pages/ImageCreate.tsx (hunk 2 above)
Comment (kitorer) [inline mobile/src/pages/ImageCreate.tsx:81]: while a image is being created, disable this button to prevent spam calls

**Review by kitorer — CHANGES_REQUESTED** [review]

File: mobile/src/pages/ImageCreate.tsx (hunk 2 above)
Comment (fabriUCF) [reply mobile/src/pages/ImageCreate.tsx:81]: The tags and search don't update as well.

**Review by fabriUCF — CHANGES_REQUESTED** [review]

File: mobile/src/pages/ImageCreate.tsx (hunk 2 above)
Comment (rob-3) [reply mobile/src/pages/ImageCreate.tsx]: @fabriUCF wdym by this?

**Review by rob-3 — APPROVED** [review]

**Review by kitorer — APPROVED** [review]

File: mobile/src/pages/ImageCreate.tsx (hunk 2 above)
Comment (danielagd) [reply mobile/src/pages/ImageCreate.tsx]: working on the issues now

Comment (fabriUCF) [reply mobile/src/pages/ImageCreate.tsx]: Oh so this essentially means, once an image is created the prompt and tags remain the same (hence the "they don't update"). I didn't know if that was intentional so I just commented it. Sorry lol I thought I dm'd Dani about it.

**Review by fabriUCF — APPROVED** [review]

File: mobile/src/pages/ImageCreate.tsx (hunk 2 above)
Comment (rob-3) [reply mobile/src/pages/ImageCreate.tsx]: Oh good idea, yeah we should fix that

**Pull request merged**